src/json-completion.ts

```diff
@@ -267,69 +302,74 @@ export class JSONCompletion {
     let value;
     let nValueProposals = 0;
     if (typeof propertySchema === "object") {
-      if (propertySchema.enum) {
-        if (!value && propertySchema.enum.length === 1) {
-          value = this.getInsertTextForGuessedValue(propertySchema.enum[0], "");
-        }
-        nValueProposals += propertySchema.enum.length;
-      }
-      if (typeof propertySchema.const !== "undefined") {
-        if (!value) {
-          value = this.getInsertTextForGuessedValue(propertySchema.const, "");
-        }
-        nValueProposals++;
-      }
       if (typeof propertySchema.default !== "undefined") {
+        console.log("default", propertySchema.default, value);
         if (!value) {
           value = this.getInsertTextForGuessedValue(propertySchema.default, "");
         }
         nValueProposals++;
-      }
-      if (
-        Array.isArray(propertySchema.examples) &&
-        propertySchema.examples.length
-      ) {
-        if (!value) {
-          value = this.getInsertTextForGuessedValue(
-            propertySchema.examples[0],
-            ""
-          );
+      } else {
```

Comment:
The tests will make us more confident with these type of changes!

Reply:
indeed! i will add tests for all of these cases before merging this PR